data4apertium/corpora/jam/uzb.txt

```diff
@@ -1,13 +1,11 @@
-Baxtiyor kichkina bola va u olti yoshda. Kichkina qiz uning singlisi u besh yoshda. Baxtiyorni kichkina kuchugi bor, kuchuk ham hozir bog'da. Kuchuk ikkita kichkina bolalar bilan o'ynashni yaxshi ko'radi. Kuchuk hozir juda hursand.
-Oygul kuchukni yaxshi ko'radimi? Yo'q, Oygulda kuchuk yo'q, unda mushuk bor. Ammo, mushuk uyda, mushuk uxlayapti.
-Ularning onasi mushuk bilan uyda, u oynadan Baxtiyor va Oygulning o'ynashini tomosha qilyapti.
-Baxtiyor ulkan daraxt oldiga yugurib bordi, u Oyguldan bekinyapti. Bilasizmi nimaga? Oygul o'tiryapti va u qo'li bilan ko'zini bekityapti. U sanayotgan paytida hech nimani ko'rmayapti. Nimaga u bunday qilyapti? Va Baxtiyor daraxtni yonida nima qilyapti?
-Bu o'yin. Oygul sanashni tugatgandan so'ng u atrofga qaraydi. U Baxtiyorni qidiryapti: U qayerga ketdi? Uni ko'ryapsanmi?
-Oygul Baxtiyor qayerdaligini bilmaydi. U kuchukdan so'rayapti: "Sen Baxtiyorni ko'rdingmi?". Ammo kuchuk gapiraolmaydi albatta! Shuning uchun Oygul savoliga hech qanday javob olmaydi. Odamlar kuchukga gapirishganda hech qachon javob olishmaydi!
-Oygul oyna orqasidagi onasini ko'rdi, uning onasi kulyapti. Oygul uni Baxtiyorni qayerga ketganini ko'rdi deb o'yladi: "Baxtiyor qayerga ketdi!", u aytdi onasiga. "Yo'q Oygul, men ayta olmayman!", onasi javob berdi. U Baxtiyor qayerdaligini bilsada, u aytgisi kelmayapti.
-Oygul bog'dan sekin yuryapti. U haliyam Baxtiyorni qidiryapti. U stul va stolni tagini qaradi, ammo Baxtiyor u yerda emas. U hamma joyni qidirib chiqdi, ammo u Baxtiyorni topa olmadi.
-Keyin u ovozni eshitdi, u ulkan daraxtni yonidan kelayotgandi. U Baxtiyor bo'lishi mumkinmi?Yana o'sha ovoz kelyapti! U diqqat bilan tingladi.Endi u uni yaxshi eshityapti. U Baxtiyor bo'lsa kerak!
-Keyin u kichkina qo'lni ko'rdi va u yaqinlashgan sari uni boshini ham ko'rdi! U kuldi va aytdi: "Men seni topdim!". Ularni ikkalasi ham baxtiyor edi va ular uyga kirishdi, endi ovqatlanish va suv ichishni vaqti keldi!
-
+1	BAXTIYOR QAYERDA?
+2	Baxtiyor va Oygul bog'da. Bugun ob-havo yaxshi, havo juda iliq. Ammo kecha juda sovuq edi! Ular o'ynay olishmadi. Baxtiyor va Oygul o'ynashni yaxshi ko'radilar, ular har doim katta hovli oldidagi bog'da birga o'ynashadi.
+3	Baxtiyor kichkina bola va u olti yoshda. Kichkina qiz uning singlisi, u besh yoshda. Baxtiyorning kichkina kuchugi bor, kuchuk ham hozir bog'da. Kuchuk bu ikki bolakaylar bilan o'ynashni yaxshi ko'radi. Kuchuk hozir juda hursand.
+4	Oygulning ham kuchugi bormi? Yo'q, Oygulda kuchuk yo'q, unda mushuk bor. Ammo, mushuk uyda, mushuk uxlayapti.
+5	Ularning onasi mushuk bilan uyda, u oynadan qaraydi va Baxtiyor bilan Oygulning o'ynayotganini ko'radi. Baxtiyor ulkan eski daraxt oldiga yugurib boradi, u Oyguldan bekinyapti. Bilasizmi nimaga? Oygul o'tiryapti va qo'li bilan ko'zini bekityapti. U hech nimani ko'rolmaydi va u sanayabdi. Nimaga u bunday qiladi? Va Baxtiyor daraxtni yonida nima qilyapti?
```

Comment:
> Could you just say "u oynadan Baxtiyor bilan Oygulning oʼynayotganini koʼradi"?

Yes, but I just wanted to make it an exact translation of the story in english, where it says: "she looks through the window and sees James and Mary playing"(two verbs were used: to look and to see).

Reply:
> Is it "koʼrolmaydi" or "koʼra olmaydi"?

Both are correct.

Reply:
> I just wanted to make it an exact translation of the story in english

Our priorities here, in terms of choices in translation, are:
1. Have the text be good Uzbek.  It shouldn't sound awkward in order to match another language (at least not one we're not doing MT with), 
2. Keep it internally consistent (if a choice is made in one place, it should be made elsewhere, and
3. Perhaps keep the structure more similar to that of other Turkic translations, since that's what we'll be testing translation between.

Does the two-verb translation meet #1?  Have you checked other Turkic versions of the text?

Reply:
> > Is it "koʼrolmaydi" or "koʼra olmaydi"?
>
> Both are correct.

Okay, but both variants are used in this story.  Choose the one you'd like the morphological transducer and MT to *create*, and go with that.  (We should of course be able to analyse both forms.)